src/UberStrok.WebServices/ItemManager.cs

```diff
@@ -22,10 +23,16 @@ public ItemManager(WebServiceContext ctx)
 
         private readonly UberStrikeItemShopClientView _items;
         private readonly WebServiceContext _ctx;
+        private readonly List<BundleView> _bundles;

```

Comment:
This is not initialized, should be an empty list
```suggestion
_bundles = new List<BundleView>();
```

Reply:
Do that somewhere in the constructor.